published/Standup/README

```diff
@@ -0,0 +1 @@
+Ellipsis will ask members of a team 4 standup questions and report on them just before a standup meeting. 
```

Comment:
Maybe we should just omit the "4" here so it's never wrong. :)

"Ellipsis will ask members of a team some standup questions…"